tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.hpp

```diff
@@ -528,6 +516,105 @@ void test_suite_for_map_on_sphere(
   test_helper(map2);
 }
 
+  /*!
+   * \ingroup TestingFrameworkGroup
+   * \brief Given a Map `map`, tests the map functions, including map inverse,
+   * jacobian, and inverse jacobian, for a series of points.
+   * These points are chosen in a right cylinder with cylindrical radius 1
+   * and I1-axis extending from -1 to +1. The
```

Comment:
sorry, `I1-axis` here is the "z-axis"? I'm hoping we can avoid some (much) of the SpEC terminology like `I1` `S2`, `B3` etc. when it's at least reasonably easy to do so :)